README.md

```diff
@@ -201,6 +201,28 @@ async def transform(msg):
     ...
 ```
 
+### Serialization
+
+Dispatch uses the [pickle] library to serialize coroutines.
+
+[pickle]: https://docs.python.org/3/library/pickle.html
+
+Serialization of coroutines is enabled by a CPython extension.
+
+The user must ensure that the contents of their stack frames are
+serializable. That is, users should avoid using variables inside
+coroutines that cannot be pickled.
+
+If a pickle error is encountered, serialization tracing can be enabled
+with the `DURABLE_TRACE=1` environment variable to debug the issue. The
```

Comment:
How about we change this variable to be named `DISPATCH_`something ?

Reply:
See 5d8f846